bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,358 @@
+- `commission-max-change-rate`: Maximum daily commission change rate
+- `min-self-delegation`: Minimum amount you must keep self-delegated
+
+If you prefer to add this manually or are having issues, another option is to 
+create a `validator.json` file manually and then paste the above json into it 
+but remember to replace all values with the actual values you want to use.
+
+## 5. Creating a Validator
+
+> ⚠️ **Important**: You will need a funded account for this step
+
+> ⚠️ **Important**: Please make sure to read through this section
+> as it might not work with your automations for creating validators.
+
+Unlike traditional Cosmos SDK chains that use the `staking` module, 
+Babylon uses the Babylon [`checkpointing`](https://docs.babylonlabs.io/docs/developer-guides/modules/checkpointing) module for validator creation and management.
+
+The creation process requires your previously generated BLS key and your 
+validator key, which should be located at `<path>/config/validator.json`, 
+where `<path>` is the `--home` directory you specified when setting up your node.
+
+The BLS key in the file is required as validators use their BLS key to sign 
+checkpoint messages, which are aggregated and submitted to Bitcoin for finality.
+
+To create your validator, run the following command:
+
+```shell 
+babylond tx checkpointing create-validator \
+    ./<home-path>/config/validator.json \
+    --chain-id bbn-test-5 \
+    --gas "auto" \
+    --gas-adjustment 1.5 \
+    --gas-prices "0.005ubbn" \
+    --from <your-key-name> \
+    --keyring-backend <keyring-backend> \
+    --home <path>
+```
+
+Parameters:
+- `--chain-id`: The network identifier
+- `--gas`: Set to "auto" to automatically calculate the gas needed
+- `--gas-adjustment`: A multiplier for the estimated gas 
+- `--gas-prices`: Transaction fee in ubbn per unit of gas
+- `--from`: Your key name or address that will sign and pay for this transaction
+- `--keyring-backend`: Specifies the keyring backend to use for key management. 
+  We have used `test` in our example.
+- `--home`: Specifies the directory where your node files will be stored 
+
+Upon successful creation, you'll be asked to approve the transaction. 
+Once approved, you'll receive a transaction hash and your validator
+operator address 
+(e.g., `bbnvaloper1qh8444k43spt6m8ernm8phxr332k85teavxmuq`).
+
+### 5.1 Verifying Validator Setup
+
+To verify your validator setup, you can use the following steps:
+
+First, get your validator's operator address using your Babylon address: 
+
+```shell
+babylond keys show <your-key-name> --address --bech val --home <path> --keyring-backend <keyring-backend>
+```
+
+For example, for the address we used above is `bbn1qh8444k43spt6m8ernm8phxr332k85teavxmuq`, 
+the operator address is `bbnvaloper1qh8444k43spt6m8ernm8phxr332k85teavxmuq`. 
+
+For the next step, we will have to wait till the beginning of the next epoch 
```

Comment:
I think we are giving the epochized validator creation explanation at the wrong place. It should not be given in the "verification of validator setup" section as many people might stop before that.

Instead, we should provide this explanation as required reading either prior or immediately after the `create-validator` command. Essentially, after the validator executes the `create-validator`, they receive an explanation as to what to expect to happen next. We can then have the "Verifying validator setup" section which just references the above explanation "e.g. as explained before, this will only work after the epoch ends..."